site/docs/v1/tech/apis/_system-configuration-response-body.adoc

```diff
@@ -500,6 +500,9 @@ For example:
 [quote]
 `America/Denver` or `US/Mountain`
 
+[field]#systemConfiguration.ssoConfiguration.deviceTrustTimeToLiveInSeconds# [type]#[Integer]# [optional]#Optional# [default]#defaults to `31,536,000` (1 year)# [since]#Available since 1.30.0#::
```

Comment:
the response should not have the default listed, and does not need `[optional]`.